js/src/components/contact-information/store-address-card.js

```diff
@@ -132,38 +108,61 @@ const StoreAddressCard = ( { showValidation = false } ) => {
 			.join( ', ' );
 
 		addressContent = (
-			<div>
+			<div className="gla-store-address-card__address">
```

Comment:
Doesn't look like it, and since the address can be missing, I've removed the redundant div and added some conditional to avoid an extra empty div.

1372d3ea